src/bp/core/routers/util.ts

```diff
@@ -158,6 +159,13 @@ export const assertSuperAdmin = (req: Request, res: Response, next: Function) =>
   next()
 }
 
+export const assertWorkspace = () => async (_req: RequestWithUser, _res: Response, next: NextFunction) => {
+  if (!_req.workspace) {
+    throw new InvalidOperationError(`Workspace is missing. Set header X-BP-Workspace`)
```

Comment:
When throwing an error in a middleware, just call next with the error instead: `next(new InvalidOp('missing'))`. This way, the error will be returned to the end user, and it will not cause unhandled exceptions in server logs

Also, _ should only be used when the variable is declared but unused, so _req should be req